### ISSUE TITLE
plantuml generator does not handle mixins  correctly and breaks diagram generation

### ISSUE THREAD

**Issue by jsnyder-csdisco** [linkml/linkml#2141]
**Describe the bug**
When using `mixins` in a schema , the generated plantuml `puml` file output generates plantuml syntax that does not parse correctly because proper spacing is missing.

**To reproduce**
Given this schema fragment:
```yaml
  Person:
    is_a: IdentifiableThing
    description: A LegalEntity that is a person.
    mixins:
     - LegalEntity
```
The generated diagram will have `galEntity` as the name and the `uses` relationship will also be "broken". The problem is that spacing between ALL the operators in plantuml syntax need spaces and there are missing.

**Expected behavior**
Diagram should support mixin notation in the diagrams.

**About your computer (if applicable, please complete the following information):**
 - OS: MacOS 
 - Browser [both  chrome, safari]
 - Version Sonoma 14.4.1 (but it is irrelevant).

**Additional context**
I have a fix for this as well as adding "ToolTip" output in plantuml text file to create hover text in SVGs for each class.

**Comment by jsnyder-csdisco** [comment]
Attached a plantuml fix. I could do a PR if there is interest.

**Comment by jsnyder-csdisco** [comment]
I can add a PR if there is interest.

**Comment by sujaypatil96** [comment]
Thanks @jsnyder-csdisco!! A PR would be very appreciated. Feel free to tag me as a reviewer after after you've made the PR.

**Comment by jsnyder-csdisco** [comment]
Fixed in 1.8.1
